### PR TITLE
Add utilities to qa macros -2

### PR DESCRIPTION
**Pull request by mmazzilli**

### CODE REVIEW

File: FirstAnalysis/HFmcvalidation.py [hunk 1]
```diff
@@ -111,10 +157,13 @@ def decayLength_reco_MC(file):
...
     hDecLenRes.Rebin(2)
     hDecLenRes.Draw()
-    printCanvas(cDecLenRes, "Resolution_DecayLength")
+    printCanvas(cDecLenRes, "BtoDForcedProduction/Resolution_DecayLength")
 
 
-fileo2 = TFile("../codeHF/CCbar_pp_AnalysisResults_O2_500files.root")
+# fileo2 = TFile("/data/Run5/plots/new_pro/AnalysisResults_O2_pp14TeV_ccbar_10M_sc3werner_20042021.root")
+fileo2 = TFile(
+    "/home/mmazzill/alice/Run3Analysisvalidation/codeHF/AnalysisResults_O2_LcpKpi_4_6.root"
```
Comment (ginnocen) [inline FirstAnalysis/HFmcvalidation.py:165]: Can you remove local paths? For the moment is fine if it refers to ../codeHF/xxx

**Comment by vkucera** [comment]
There should be no hard-coded paths to temporary files in the code. It makes it unportable.
Take the file path from a command line argument and provide the path on the machine in a documentation (Dropbox, TWiki, etc.).

**Comment by vkucera** [comment]
@mmazzilli Any update on this?

**Comment by vkucera** [comment]
@mmazzilli What is the status?

**Comment by vkucera** [comment]
> @mmazzilli What is the status?

?

**Pull request closed without merging**